Shared/sdk/SString.h

```diff
@@ -150,15 +150,15 @@ class TSplitString : public std::vector<const CHAR_TYPE*>
             if (ulPos == STRING_TYPE::npos || (uiMaxAmount > 0 && uiMaxAmount <= this->size() + 1))
             {
                 if (ulCurrentPoint <= strInput.length())
-                    push_back(&buffer[ulCurrentPoint]);
+                    this->push_back(&buffer[ulCurrentPoint]);
                 break;
             }
-            push_back(&buffer[ulCurrentPoint]);
+            this->push_back(&buffer[ulCurrentPoint]);
             buffer[ulPos] = 0;
             ulCurrentPoint = ulPos + strDelim.length();
         }
         while (this->size() < uiMinAmount)
-            push_back(&buffer[iInputLength]);
+            this->push_back(&buffer[iInputLength]);
```

Comment:
Readable though pointless

Reply:
In C++20 without 'this' caused an error

Reply:
What error?

Reply:
Can't find push_back

Reply:
Check the full logs. If it worked before then it should work now

Reply:
Error C3861

Reply:
I meant full logs, not just the error code.
Maybe the call is ambigous (which it shouldnt be)?

Why do you even change Shared code?

Reply:
Error C3861	push_back: identifier not found Client Core

Reply:
@TracerDS Can I close this?

Reply:
yes

Reply:
You could try with `using vector::push_back;`